packages/gcp-logging/test/index.ts

```diff
@@ -77,7 +166,7 @@ test('effect versions after span log changes', () => {
   ])
 })
 
-test.skip('logging - should be ok', () => {
```

Comment:
@alex-dixon this test now runs, I updated it. Do you mind checking that the traces/spans are correct?